actions/install/action.yml

```diff
@@ -0,0 +1,17 @@
+name: setup
+description: 'Installs SLSA verifier and adds it to your PATH'
+branding:
+  icon: 'package'
+  color: 'blue'
+inputs:
+  verifier-release:
+    description: 'slsa-verifier release version to install'
+    required: false
+    default: 'v1.3.0'
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        ${{ github.action_path }}/installer.sh ${{ inputs.verifier-release }}
```

Comment:
always use env variables when using variables:
```
env:
  PATH: "${{ github.action_path }}"
run: |
   bla "$PATH"
```

In this case the user would just hack themselves so it does not matter, but it's good practice to do it to learn the habit :)

Reply:
Done